components/TaskContainer.vue

```diff
@@ -81,6 +104,7 @@ export default {
     },
     addTask() {
       this.$store.dispatch('todos/addTask', this.taskDescription);
```

Comment:
Can we await this if this returns promise? If it doesn't, ignore this.

Reply:
It does not return a promise just yet